backend/src/router/authRoutes.ts

```diff
@@ -14,4 +14,9 @@ router.post("/signup/counselor", signUpCounselor);
 router.post("/signup/reviewer", signUpReviewer);
 router.post("/signup/admin", signUpAdmin);
 
+router.post("/recover", recoverPassword);
+router.get("/reset/:token", redirectReset);
+router.post("/reset/:token", [
+    check('confirmPassword', 'Passwords do not match').custom((value, { req }) => (value === req.body.password)),
```

Comment:
just put this in a func in controllers as well

Reply:
cant